sdk/core/core/test/test_http_response_parser.h

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
can we also put through headers from an HTTP fuzzer?

Reply:
Sure, could you give a good example how we can get them? In the future, we may also have a non-deterministic stress-test from some HTTP fuzzer app.